### PR TITLE
improvement: make `Duplicated node_id` error more actionable

### PR DESCRIPTION
**Pull request by davidxia**
by suggesting using `.with_id()`.

### CODE REVIEW

**Review by chongkong — APPROVED** [review]

**Comment by davidxia** [comment]
Thanks, does a bot pick this commit up for merging later?

File: tfx/orchestration/pipeline.py [hunk 1]
```diff
@@ -376,7 +376,7 @@ def _set_components(self, components: List[base_node.BaseNode]) -> None:
       if component.id in node_by_id:
         raise RuntimeError(
             f'Duplicated node_id {component.id} for component type'
-            f'{component.type}.')
+            f'{component.type}. Try setting a different node_id using `.with_id()`.')
```
Comment (jiyongjung0) [inline tfx/orchestration/pipeline.py:379]: Could you break this line to make it fit in 80 column?

We need to follow the [style guide](https://google.github.io/styleguide/pyguide.html#32-line-length).

Comment (davidxia) [reply tfx/orchestration/pipeline.py]: Thanks, updated

**Review by jiyongjung0 — APPROVED** [review]
Thanks!

**Pull request merged**